Cargo.toml

```diff
@@ -7,7 +7,6 @@ edition = "2018"
 [dependencies]
 mongodb = { version = "1.0.0" }
 tokio = "0.2.19"
-bson = "1.0.0"
```

Comment:
The driver re-exports `bson` now, so no need to add it as an explicit dependency.